rpi_backlight/__init__.py

```diff
@@ -2,7 +2,7 @@
 import time
 from contextlib import contextmanager
 from enum import Enum
-from os import PathLike
+from os import PathLike, path
```

Comment:
Please don't use old-style `os.path` APIs, use `pathlib.Path` instead

Reply:
> These code changes no longer belong together, either squash the history into two distinct, clean commits, or make a second PR for the path changes if you can't do that.

Ive moved the new path stuff into https://github.com/linusg/rpi-backlight/pull/50
This PR can remain the old stuff as it has all the context

Reply:
Unused import remains

Reply:
Whoops sorry! Fixed in https://github.com/p1r473/rpi-backlight/commit/bf2d2e990f74f1c4e030ccef33c876283a0e89dd